pkg/ingester/ingester_v2.go

```diff
@@ -79,6 +79,7 @@ type TSDBState struct {
 
 // NewV2 returns a new Ingester that uses prometheus block storage instead of chunk storage
 func NewV2(cfg Config, clientConfig client.Config, limits *validation.Overrides, registerer prometheus.Registerer) (*Ingester, error) {
+	util.WarnExperimentalUse("TSDB storage engine")
```

Comment:
We'll get rid of the TSDB naming and call it "blocks storage". The doc mostly calls it "blocks storage", while the config still mention it "TSDB".

```suggestion
	util.WarnExperimentalUse("Blocks storage engine")
```